frontend/templates/list-annotation-category.tmpl

```diff
@@ -0,0 +1,71 @@
+{{!-- Skalenwerte zuerst --}}
+{{#each content}}
+    {{#ifeq type "scaling"}}
+        <span class="scaling">
+            {{#with (withScaleValue value.scaling) as |scaleValue|}}
+                <span class="read-only" title="{{scaleValue.name}}" >{{scaleValue.name}}</span>
+            {{/with}}
+        </span>
+    {{/ifeq}}
+{{/each}}
+
+{{#each content}}
+
+    {{!-- Annotationen mit Kennzeichen und Skalenwert --}}
+    {{#ifeq type "scaling"}}
```

Comment:
This whole `ifeq` is indented weirdly. Also as I noted elsewhere, I would indent the contents of "block helpers" as well, so

```
{{ifeq ...}}
    <span>...</span>
{{/ifeq}}
```

as opposed to 
```
{{ifeq ...}}
<span>...</span>
{{/ifeq}}
```